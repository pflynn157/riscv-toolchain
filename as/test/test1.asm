add x0, x1, x2
add x1, x1, x2
add x2, x1, x2
add x3, x1, x2
add x4, x1, x2
add x5, x1, x2
add x6, x1, x2
add x7, x1, x2
add x8, x1, x2
add x9, x1, x2
add x10, x1, x2
add x11, x1, x2
add x12, x1, x2
add x13, x1, x2
add x14, x1, x2
add x15, x1, x2
add x16, x1, x2
add x17, x1, x2
add x18, x1, x2
add x19, x1, x2
add x20, x1, x2
add x21, x1, x2
add x22, x1, x2
add x23, x1, x2
add x24, x1, x2
add x25, x1, x2
add x26, x1, x2
add x27, x1, x2
add x28, x1, x2
add x29, x1, x2
add x30, x1, x2
add x31, x1, x2
