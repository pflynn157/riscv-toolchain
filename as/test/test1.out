00208033
002080b3
00208133
002081b3
00208233
002082b3
00208333
002083b3
00208433
002084b3
00208533
002085b3
00208633
002086b3
00208733
002087b3
00208833
002088b3
00208933
002089b3
00208a33
00208ab3
00208b33
00208bb3
00208c33
00208cb3
00208d33
00208db3
00208e33
00208eb3
00208f33
00208fb3
